app/assets/javascripts/application.js

```diff
@@ -14,3 +14,8 @@
 //= require jquery_ujs
 //= require turbolinks
 //= require_tree .
+$(document).ready(function(){
```

Comment:
This code should probably go in its own JS file. It's best not to write JS code in this file, and to just use it to 'require' other JS things since its a 'manifest' file.